frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoService.java

```diff
@@ -0,0 +1,89 @@
+package ch.shiftcrypto.bitboxapp;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.Service;
+import android.content.Intent;
+import android.graphics.drawable.Icon;
+import android.os.Binder;
+import android.os.Build;
+import android.os.IBinder;
+
+import androidx.core.app.NotificationCompat;
+
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+import goserver.GoAPIInterface;
+import goserver.GoEnvironmentInterface;
+import goserver.Goserver;
+
+public class GoService extends Service {
+    // Binder given to clients
+    private final IBinder binder = new GoServiceBinder();
+
+    private boolean started = false;
+
+    private Lock startedLock = new ReentrantLock();
+
+    private final String channelId = "21";
+
+    private final int notificationId = 8;
+
+    @Override
+    public void onCreate() {
+        Util.log("GoService onCreate()");
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationChannel channel = new NotificationChannel(
+                    channelId,
+                    "BitBoxApp",
+                    NotificationManager.IMPORTANCE_LOW
+            );
+            channel.setDescription("BitBoxApp service notification channel");
+            NotificationManager manager = getSystemService(NotificationManager.class);
+            manager.createNotificationChannel(channel);
+        }
+
+        Notification notification = new NotificationCompat.Builder(this, channelId)
+                .setContentTitle("BitBoxApp backend service")
+                .setContentText("Running in the background")
```

Comment:
I think this could lead users to question what this is doing (background activity can sound a bit scary).

I wonder how we could rephrase it. Maybe "Running in the background to ensure connection stability" (this will also helps with Electrum connection stability). Wdyt?